js/components/base-components/listItem.js

```diff
@@ -0,0 +1,51 @@
+var React = require('react');
+var Icon = require('../icon');
+var classnames = require('classnames');
+
+var listItem = React.createClass({
+  handleSelect: function () {
+    this.props.handleSelect(this.props.id);
+  },
+
+  render: function () {
+    var classes = {
+      listItem: 'oo-list-item',
+      icon: 'icon',
+      text: 'text'
+    };
+
+    if (this.props.selected) {
+      for (var key in classes){
+        classes[key] = classes[key] + ' select';
+      }
+    }
+
+    return (
+      <div onClick={this.handleSelect} className={classes.listItem}>
+        <span className={classes.icon}> X </span>
+        {/*<Icon skinConfig={this.props.skinConfig} icon="selectedIcon" className={classes.icon} />*/}
```

Comment:
If this is a work in progress it might be better to just remove this for now, otherwise we might forget and leave this comment in there.